util/src/test/java/com/google/cloud/hadoop/util/HttpTransportFactoryTest.java

```diff
@@ -106,6 +116,77 @@ public void testParseProxyAddressWithPath() throws Exception {
         .contains("Invalid proxy address 'foo-host:1234/some/path'.");
   }
 
+  @Test
```

Comment:
can we improve the tests to include 
1. verify all methods are wired for passed ssl factory instead just one 
2. veridy socket keep alive set for all methods

Reply:
Will do